src/Sylius/Bundle/OrderBundle/Controller/AdjustmentController.php

```diff
@@ -53,4 +53,33 @@ public function lockAction(Request $request, $id)
 
         return $this->redirectHandler->redirectTo($order);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createNew()
+    {
+        if (null === $orderId = $this->getRequest()->get('orderId')) {
+            throw new NotFoundHttpException('No order id given.');
```

Comment:
just for consistency, isn't it better to use 'createNotFoundException' method? In that case we don't need to have 'NotFoundHttpException' here.